### DISCUSSION TITLE
How to install pdm-venv with pdm installed via windows installer?

### DISCUSSION THREAD

**Discussion by jamestansx** [pdm-project/pdm#540] [Q&A] [answered]
Hi, I would like to ask how to install [pdm-venv](https://pdm-project.github.io/pdm-venv/) if pdm is installed via 
````
(Invoke-WebRequest -Uri https://raw.githubusercontent.com/pdm-project/pdm/main/install-pdm.py -UseBasicParsing).Content | python -
````
command on Windows?

Because in the doc, it stated that 

> Note that pdm-venv must be installed to the same environment as pdm.

**Comment by frostming** [comment, marked as answer]
On PDM 1.6.4+, you can install the plugin directly by:
```
$ pdm plugin add pdm-venv
```

**Reply by jamestansx** [reply]
Oh I see. Thanks for the reply!
